policy/README.md

```diff
@@ -227,6 +236,7 @@ An individual `Rule` object is defined by the following fields:
 | `geographies`      | UUID[]                      | Required   | List of Geography UUIDs (non-overlapping) specifying the covered geography |
 | `states`           | `{ state: event[] }`        | Required   | [Vehicle state][vehicle-states] to which this rule applies.  Optionally provide a list of specific [vehicle events][#vehicle-events] as a subset of a given status for the rule to apply to. An empty list or `null`/absent defaults to "all". |
 | `rule_units`       | enum                        | Required   | Measured units of policy (see [Rule Units](#rule-units)) |
+| `accessibility_options` | [AccessibilityOption][accessibility-options][] | Applicable vehicle [accessibility options][accessibility-options], default any (or none) |
```

Comment:
Yep... I think for now I'm going to keep it here. Policy right now considers _events_, and device properties, but technically doesn't relate to anything else. We'll have to think about how to extend policy so it can apply to Trips as a first-class entity as well. But that may be an iteration after Taxi is initially introduced, too many changes at the same time.

Reply:
Okay, I agree with that thinking and for deferring this until later. This is the least important/pressing of those three policies, but I do expect it'll be something that will be required at some point.

Another possible way to look at this could still be _event-based_ policies, but ones that are conditional on certain trip metadata, e.g. `trip_end` allowed in a certain area only if accompanied by trip metadata that includes `accessibility_options: ['wheelchair_accessible']`